v22.1/add-region.md

```diff
@@ -132,5 +132,8 @@ SHOW REGIONS FROM DATABASE foo;
 - [`SET PRIMARY REGION`](set-primary-region.html)
 - [`DROP REGION`](drop-region.html)
 - [`SHOW REGIONS`](show-regions.html)
+- [`ADD SUPER REGION`](add-super-region.html)
+- [`DROP SUPER REGION`](drop-super-region.html)
+- [`SHOW SUPER REGIONS`](show-super-regions.html)
```

Comment:
There was also ALTER super region support added - just wanted to make sure that this is also on your radar but it can be added afterwards.
https://github.com/cockroachdb/cockroach/pull/78367